static/third/bootstrap-tooltip/tooltip.css

```diff
@@ -190,3 +190,9 @@
   border-left-color: rgba(0, 0, 0, 0.25);
   border-width: 11px 0 11px 11px;
 }
+
+@media (hover: none) {
```

Comment:
@ligmitz can you add `https://drafts.csswg.org/mediaqueries-4/#mf-interaction` link with some comment here. Other than that it looks and works great for me.

Reply:
Sure !

Reply:
Can you move this to static/styles/zulip.css in the main `tooltip` block there?  We generally prefer to avoid avoiding third-party code, since it makes upgrading said code much harder.

Reply:
Okay